stories/tokens/colours/colours.scss

```
@@ -0,0 +1,47 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../../../packages/bpk-mixins/index.scss';
+
+.bpk-colour-container {
+  width: bpk-spacing-md() * 19;
+  height: bpk-spacing-xxl() * 5;
+  margin: bpk-spacing-base();
+  padding: bpk-spacing-base();
+  align-items: center;
+  border-radius: $bpk-border-radius-md;
+  background-color: $bpk-surface-default-day;
+  box-sizing: border-box;
+
+  @include bpk-border-sm($bpk-line-day);
+}
+
+.bpk-colour-swatch {
+  height: bpk-spacing-md() * 15;
+  border-radius: $bpk-border-radius-sm;
+
+  @include bpk-border-sm($bpk-line-day);
+}
+
+.bpk-colour-name {
```

Code should use American `color`
```suggestion
.bpk-color-name {
```